clients/vulnerable-code/src/main/kotlin/VulnerableCodeService.kt

```diff
@@ -40,7 +40,7 @@ interface VulnerableCodeService {
         /**
          * The mapper for JSON (de-)serialization used by this service.
          */
-        val JSON_MAPPER = JsonMapper().setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE)
+        private val JSON_MAPPER = JsonMapper().setPropertyNamingStrategy(PropertyNamingStrategies.SNAKE_CASE)
```

Comment:
Please keep this as-is [as already discussed in one of your previous code inspection fix PRs](https://github.com/oss-review-toolkit/ort/pull/3636#discussion_r576092369).

Reply:
Damn, my brain refuses to remember these kind of edge cases. Done.